openedx/core/djangoapps/notifications/base_notification.py

```diff
@@ -179,6 +180,24 @@
         'email_template': '',
         'filters': [FILTER_AUDIT_EXPIRED_USERS_WITH_NO_ROLE]
     },
+    'ora_staff_notification': {
```

Comment:
Should we have a better name for this? What if we want to include more roles to the audience?

Reply:
This notification is for grading submissions and is targeted towards staff and instructors.  So no need to update.